doc/make.jl

```diff
@@ -113,7 +113,7 @@ const PAGES = [
 ]
 
 makedocs(
-    build     = "_build/html/en",
+    build     = joinpath(pwd(), "_build/html/en"),
```

Comment:
The `build` is relative to `root`, which defaults to the script's directory (`Base.source_dir()`), which will be in the source not build directory.